### PR TITLE
Implement docker for dev purpose

### PR DESCRIPTION
**Pull request by kistora**
### Description of your PR

Implement docker using docker-compose.
I don't want do loose time configuring the project :wink: 
Note that this PR won't impact already configured project on other developpeur laptop.

### Pre-review TODO

- [x] PR is rebased on top of `origin/master` 
- [ ] PR provide new tests for new behaviors
- [ ] PR doesn't introduce commented out code
- [x] PR has been tested locally on the main use cases
- [x] Documentation have been updated if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,10 +98,48 @@ you can install these plugins to watch linter errors.
...
+
+P.S.: The file `.proxy.conf.json` is already ignored by git.
+
+### Lunch containers!
+
+Then: `docker-compose up`
+
+`docker-compose` will create 4 different containers:
+- `k-app-mongo`: a mongodb server
+- `k-app-mysql`: a mongodb mysql
```
Comment (Embraser01) [inline README.md]:
```suggestion
- `k-app-mysql`: a mysql server
```

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      - .:/k-app
+    depends_on:
+      - mysql
+      - mongo
+    entrypoint:
+      - /bin/sh
+      - -c
+      - |
+        cd k-app
+        echo "Install yarn with npm"
```
Comment (Embraser01) [inline docker-compose.yaml]: Yarn is already packaged with node docker image so no need to install it :wink:

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      - .:/k-app
+    depends_on:
+      - api
+    entrypoint:
+      - /bin/sh
+      - -c
+      - |
+        cd k-app
+        echo "Install yarn with npm"
+        npm i -g yarn
```
Comment (Embraser01) [inline docker-compose.yaml]: Same here

File: docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+version: '3.1'
+services:
+  api:
+    image: node
```
Comment (Embraser01) [inline docker-compose.yaml]: We should maybe add a version number to node image (at least for the API)?

Comment (kistora) [reply docker-compose.yaml]: Yes you are right but which one ?

Comment (Embraser01) [reply docker-compose.yaml]: LTS should work

File: README.md [hunk 5]
```diff
@@ -59,11 +59,20 @@ Some of them are used only in production, others are used only in development.
...
 
 P.S.: The file `.env` is already ignored by git.
 
+#### `proxy.conf.json`
+
+As the project could be used in two different dev env, we need two different configurations for the proxy of the front.
+
+Here, we want to use the docker configuration. So just copy the `local.proxy.conf.json` file using: `cp local.proxy.conf.json proxy.conf.json`
+
+P.S.: The file `.proxy.conf.json` is already ignored by git.
```
Comment (Embraser01) [inline README.md]:
```suggestion
P.S.: The file `proxy.conf.json` is already ignored by git.
```

File: README.md [hunk 6]
```diff
@@ -89,10 +98,48 @@ you can install these plugins to watch linter errors.
...
+
+P.S.: The file `.env` is already ignored by git.
+
+#### `proxy.conf.json`
+
+As the project could be used in two different dev env, we need two different configurations for the proxy of the front.
+
+Here, we want to use the docker configuration. So just copy the `docker.proxy.conf.json` file using: `cp docker.proxy.conf.json proxy.conf.json`
+
+P.S.: The file `.proxy.conf.json` is already ignored by git.
```
Comment (Embraser01) [inline README.md]:
```suggestion
P.S.: The file `proxy.conf.json` is already ignored by git.
```

**Comment by kistora** [comment]
May I merge it ? @Embraser01

**Comment by Embraser01** [comment]
👍

**Comment by kistora** [comment]
image
@Embraser01 ?

**Comment by Embraser01** [comment]
Forgot to remove the restriction, it's good now :wink:

**Pull request merged**